fastlane/Fastfile

```diff
@@ -9,7 +9,8 @@ platform :ios do
       scheme: 'WordPressAuthenticator',
       prelaunch_simulator: true,
       buildlog_path: File.join(__dir__, '.build', 'logs'),
-      derived_data_path: File.join(__dir__, '.build', 'derived-data')
+      derived_data_path: File.join(__dir__, '.build', 'derived-data'),
+      output_style: 'raw'
```

Comment:
This change was for debugging, wasn't it? It should be okay to delete it now?

Reply:
Yes. Great catch, thank you! 🙇‍♂️

Reply:
Addressed by dropping the commit via interactive rebase 👌